sycl/source/handler.cpp

```diff
@@ -111,9 +111,10 @@ bool handler::isStateExplicitKernelBundle() const {
 std::shared_ptr<detail::kernel_bundle_impl>
 handler::getOrInsertHandlerKernelBundle(bool Insert) const {
   if (!MImpl->MKernelBundle && Insert) {
-    MImpl->MKernelBundle =
-        detail::getSyclObjImpl(get_kernel_bundle<bundle_state::input>(
-            MQueue->get_context(), {MQueue->get_device()}, {}));
+    auto Ctx = (MGraph) ? MGraph->getContext() : MQueue->get_context();
+    auto Dev = (MGraph) ? MGraph->getDevice() : MQueue->get_device();
```

Comment:
Redundant brackets here too, like in Ben's comment below:
https://github.com/intel/llvm/pull/11556/files#r1365333849